tests/lib/rules/no-assert-ok.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @fileoverview Forbid the use of assert.ok/notOk and suggest other assertions.
+ * @author Kevin Partington
```

Comment:
You should change this line to reflect your authorship. :smile: (You can use your real name or your GitHub name, it doesn't matter to me.)